app.js

```diff
@@ -1,14 +1,25 @@
-const {getCategories, getReviews}=require('./controllers/controller')
+const {getCategories, getReviews, getReviewByID}=require('./controllers/controller')
 const express=require('express')
 const app=express()
 
 app.get('/api/categories', getCategories)
 
 app.get('/api/reviews', getReviews)
 
+app.get('/api/reviews/:review_id', getReviewByID)
+
+app.use((err,req,res,next)=>{
```

Comment:
Ideally we should have two separate middleware functions set up here, one for custom errors and one for the sql error when it returns a bad request.